lib/layout/dummyLayoutV1.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * dummyLayoutV1.js
+ *
+ * this dumps a fake graph
+ *
+ * @author John O'Brien. 
+ */
+
+// Create a graph instance.
+
+
+function buildGraph() {
```

Comment:
This should probably get moved into some kind of object. How does "DummyGraphStrategy" sound? Basically it would have a single "buildGraph" method. The other graph strategies would also have the same method name but in a different class. That way, the strategies are truly interchangeable.